nservicebus/extending/queuecreation.md

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
Took that over from the previous commits. Haven't thought about that. Where do you suggest to put it?